tests/support/cluster_util.tcl

```diff
@@ -253,7 +253,7 @@ proc start_cluster {masters replicas options code {slot_allocator continuous_slo
 
     # Configure the starting of multiple servers. Set cluster node timeout
     # aggressively since many tests depend on ping/pong messages. 
-    set cluster_options [list overrides [list cluster-enabled yes cluster-ping-interval 100 cluster-node-timeout 3000]]
+    set cluster_options [list overrides [list cluster-enabled yes cluster-ping-interval 100 cluster-node-timeout 3000 cluster-slot-stats-enabled yes]]
```

Comment:
Why change this globally for all cluster tests? To make sure have good coverage on that assert?

Reply:
To have good coverage on that assert was my main goal with this change, yes.